infra/fastly/terraform/dl.k8s.io/services.tf

```diff
@@ -0,0 +1,89 @@
+resource "fastly_service_vcl" "files" {
+  name     = "cdn.dl.k8s.io"
+  activate = true
+
+  domain {
+    name = "cdn.dl.k8s.io"
+  }
+
+  backend {
+    name             = "GCS"
+    auto_loadbalance = false
+    shield           = "bfi-wa-us"
```

Comment:
> [shield](https://registry.terraform.io/providers/fastly/fastly/latest/docs/resources/service_vcl#shield) (String) The POP of the shield designated to reduce inbound load

is not kind of contradictory for using a cdn?

Reply:
Yeah. we use this ensure objects are pulled through Fastly network.